aten/src/ATen/native/native_functions.yaml

```diff
@@ -1261,6 +1261,18 @@
 - func: cosine_similarity(Tensor x1, Tensor x2, int64_t dim=1, double eps=1e-8) -> Tensor
   variants: function
 
+- func: to_squareform(Tensor self) -> Tensor
+  python_module: nn
```

Comment:
these shouldn't go int the nn module, this function doesn't really have anything to do with neural networks specifically.